src/dev/activation.ts

```diff
@@ -157,7 +157,12 @@ class VirtualObjectFile implements FileProvider {
             const value = (await this.storage.get(key)) ?? ''
             return JSON.stringify(JSON.parse(value), undefined, 4)
         } else {
-            return JSON.stringify(this.storage.get(key, {}), undefined, 4)
+            if (key !== '') {
+                return JSON.stringify(this.storage.get(key, {}), undefined, 4)
+            }
+            // lol hax
+            // as of November 2023, all of a memento's properties are stored as property `f` when minified
+            return JSON.stringify((this.storage as any).f, undefined, 4)
```

Comment:
If we want to get smarter about this, we can iterate through the properties and look for an obvious tell, like `object.hasOwnProperty('isExtensionFirstUse')` and return that.

We can also return a READONLY editor for this base case.

I'm not going to think too hard about this now because this is dev mode only.